src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -864,11 +907,18 @@ def execute_task(full_fuzzer_name, job_type):
     raise CorpusPruningException(
         'Failed to set up fuzzer %s.' % fuzz_target.engine)
 
+  cross_pollination_method = choose_cross_pollination_strategy(full_fuzzer_name)
+  tag = None
+  if cross_pollination_method == TAGGED_METHOD:
+    tag = random.choice(
+        corpus_tagging.get_fuzz_target_tags(full_fuzzer_name)).tag
+
```

Comment:
actually, it might be even better to give up on the `TAGGED_METHOD`. Let's say a fuzz target has two tags, one is used by other targets and the other one is unique. The call on line 880 can return some targets, while here we'll choose a tag that is unique for the current fuzzer and will not actually do any tagged cross pollination

Reply:
Done